common/determined_common/experimental/checkpoint/_torch.py

```diff
@@ -10,7 +10,7 @@
 def load_model(
     ckpt_dir: pathlib.Path, metadata: Dict[str, Any], **kwargs: Any
 ) -> Union[PyTorchTrial, torch.nn.Module]:
-    checkpoint = torch.load(ckpt_dir.joinpath("state_dict.pth"), map_location="cpu")  # type: ignore
+    checkpoint = torch.load(ckpt_dir.joinpath("state_dict.pth"), **kwargs)  # type: ignore
```

Comment:
question: does load default to `map_location="cpu"`

Reply:
This defaults to whatever the weights were saved from.

Reply:
I see. That's why we mapped to the CPU for testing.